notebook/static/notebook/js/kernelselector.js

```diff
@@ -265,15 +266,19 @@ define([
             );
         });
         
+        var no_kernel_msg = i18n.sprintf(i18n._("Could not find a kernel matching %s. Please select a kernel:"),
+        		(data.selected.display_name || data.selected.name))
         var body = $("<form>").addClass("form-inline").append(
-            $("<span>").text(
-                "I couldn't find a kernel matching " + (data.selected.display_name || data.selected.name) + "." +
-                " Please select a kernel:"
-            )
+            $("<span>").text(no_kernel_msg)
         ).append(select);
+
+        // This statement is used simply so that message extraction
+        // will pick up the strings.  The actual setting of the text
+        // for the button is in dialog.js.
+        var button_labels = [ i18n._("Continue Without Kernel"), i18n._("Set Kernel"), i18n._("OK") ];
         
         dialog.modal({
-            title : 'Kernel not found',
+            title : i18n._('Kernel not found'),
             body : body,
             buttons : {
                 'Continue Without Kernel' : {
```

Comment:
I'm guessing that because JS syntax allows `{a: 1}`, where the key becomes the string 'a', it can't also allow `{i18n._('continue'): 1}` to translate the key in the dictionary in place? In Python there would be no problem witht that. Just an annoyance.